app/build.gradle

```diff
@@ -201,6 +201,8 @@ dependencies {
     implementation "com.nononsenseapps:filepicker:4.2.1"
 
     implementation "ch.acra:acra-core:5.5.0"
+    compileOnly "com.google.auto.service:auto-service-annotations:1.0-rc7"
+    kapt "com.google.auto.service:auto-service:1.0-rc7"
```

Comment:
Could you move this up to the other `com.google` dependencies? And could you put the version in `ext` instead? That's what we do if we depend on multiple parts of the same library that thus needs the same version.

Reply:
> Could you move this up to the other com.google dependencies?

I put them here since they are related to ACRA, but ok

> And why depend on a RC?

Apparently ACRA's (probably a little bit outdated) documentation suggests `1.0-rc6`, so I chose the latest RC, `1.0-rc7`. The version suggested in e.g. google/auto#665 (again, outdated) is also an rc, and in the [releases page](https://github.com/google/auto/releases) there is no AutoService version which is not an RC (with `rc1` dating back to 2014).

Reply:
Done

Reply:
Done

Reply:
>I put them here since they are related to ACRA, but ok

No, they're not related to ACRA, we just currently happen to only use it in that code.

>Apparently ACRA's (probably a little bit outdated) documentation suggests 1.0-rc6, so I chose the latest RC, 1.0-rc7. The version suggested in e.g. google/auto#665 (again, outdated) is also an rc, and in the releases page there is no AutoService version which is not an RC (with rc1 dating back to 2014).

I found out about there not being a non-RC version just after I commented, so I think that's OK.